doxia-modules/doxia-module-markdown/src/test/java/org/apache/maven/doxia/module/markdown/MarkdownParserTest.java

```diff
@@ -798,4 +800,24 @@ public void testQuoteVsApostrophe() throws Exception {
                 "This apostrophe isn't a quote." + "This \u2018quoted text\u2019 isn't surrounded by apostrophes.",
                 content.toString());
     }
+
+    @Override
+    protected void assertEventPrefix(Iterator<SinkEventElement> eventIterator) {
+        assertSinkStartsWith(eventIterator, "head", "head_", "body");
+    }
+
+    @Override
+    protected void assertEventSuffix(Iterator<SinkEventElement> eventIterator) {
+        assertSinkEquals(eventIterator, "body_");
+    }
+
+    @Override
+    protected String getVerbatimSource() {
+        return null; // not supported in MD
```

Comment:
Indeed, CommonMark does not support this. Eventually a link to https://spec.commonmark.org/0.31.2/#fenced-code-blocks might make sense?

Reply:
Not sure fenced code blocks are related. There is also https://spec.commonmark.org/0.31.2/#indented-code-block and both really refer to code only as there is no verbatim without code semantics.

Reply:
This isn't what a meant. I meant just add the link I have provided and explain why we return null since this is not a missing/broken test.